Source/App/HeadlessExportMain.cpp

```
@@ -47,6 +50,41 @@ void printUsage(const char* programName) {
     std::cout << "  --help                Show this help message\n";
 }
 
+const char* requireValue(int argc, char* argv[], int& index, const std::string& option) {
```

Cppcheck (2.20.0)

[style] The function 'toString' is never used.

(unusedFunction)